tests/test_understat.py

```diff
@@ -90,42 +90,36 @@ async def test_get_league_results_with_options(self, event_loop, understat):
             "epl", 2018, isResult=True)
         assert results_with_option == results_without_option
 
-    async def test_get_league_fixtures(self, event_loop, understat):
+    async def test_get_league_fixtures(self, loop, understat):
         for league in leagues:
             fixtures = await understat.get_league_fixtures(league, 2018)
             assert isinstance(fixtures, list)
 
         for fixture in fixtures:
             assert not fixture["isResult"]
 
-    async def test_get_league_fixtures_with_options(self, event_loop, understat):
-        results = await understat.get_league_fixtures("epl", 2022, {
```

Comment:
In my opinion, this test should be outright deleted as it will always fail at some random point in the future.